_tour/basics.md

```diff
@@ -16,7 +16,7 @@ In this page, we will cover the basics of Scala.
 
 You can run Scala in your browser with _ScalaFiddle_. This is an easy, zero-setup way to experiment with pieces of Scala code:
```

Comment:
```suggestion
You can run Scala in your browser with _Scastie_. This is an easy, zero-setup way to experiment with pieces of Scala code:
```

Reply:
We should also update the text in the other languages.

Reply:
Thanks, updated. I guess once merged we can close this issue https://github.com/scala/docs.scala-lang/issues/2417 too